content/docs/start/experiments.md

```diff
@@ -211,12 +211,257 @@ scores.json  auc       0.61314  0.07139
 And finally, we can compare `ROC` curves with a single command!
 
 ```dvc
+$ dvc plots modify prc.json -x recall -y precision
+$ git commit -am "Update plots format"
+
 $ dvc plots diff
 file:///Users/dvc/example-get-started/plots.html
 ```
 
 ![](/img/plots_roc_get_started.svg)
 
+<details>
+
+### 💡 Expand to see what plots modify does under the hood
+
+[`dvc plots`](https://dvc.org/doc/command-reference/plots#plots) is a flexible visualisation tool for your continuous metrics. [`dvc plots modify`](https://dvc.org/doc/command-reference/plots/modify), in the example above, sets what to use as an X-axis what to plot on Y-axis of a visualisation when you run `dvc plots diff` later on.
```

Comment:
Why does the GH diff here looks like you added over 200 lines to this doc? Something is wrong. Please double check the HTML and formatting.

Reply:
See https://dvc.org/doc/user-guide/contributing/docs

Reply:
I see this was already reported by Ivan. Moved to https://github.com/iterative/dvc.org/pull/1514#discussion_r448542869. Resolving here.